### PR TITLE
ss-127: add modal to add transaction

### PR DESCRIPTION
**Pull request by Gandrejko**
image
image

### CODE REVIEW

**Review by anton-otroshchenko — DISMISSED** [review]
Resolve conflicts

**Comment by nikita-remeslov** [comment]
Conflicts

File: frontend/src/bundles/common/components/transaction-modal/transaction-image.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            {file ? (
+                <img
+                    className={styles.image}
+                    src={URL.createObjectURL(file)}
+                    alt="transaction"
+                />
+            ) : (
+                <FontAwesomeIcon icon={faCamera} />
+            )}
+            <input
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/transaction-modal/transaction-image.tsx:38]: we have a common component

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/transaction-modal/transaction-image.tsx (hunk 1 above)
Comment (nikita-remeslov) [reply frontend/src/bundles/common/components/transaction-modal/transaction-image.tsx:38]: Not resolved

**Review by anton-otroshchenko — APPROVED** [review]

File: frontend/src/bundles/common/components/transaction-modal/transaction-image.tsx (hunk 1 above)
Comment (Evcat6) [reply frontend/src/bundles/common/components/transaction-modal/transaction-image.tsx:38]: Could you use `<Input/>` component from `bundles/common/components`?

**Review by Evcat6 — CHANGES_REQUESTED** [review]

**Review by Evcat6 — APPROVED** [review]

**Review by anton-otroshchenko — APPROVED** [review]

**Review by Evcat6 — APPROVED** [review]

File: frontend/src/bundles/common/components/transaction-modal/transaction-image.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    const inputReference = useRef<HTMLInputElement>(null);
+
+    const handleClick = useCallback((): void => {
+        if (inputReference.current) {
+            inputReference.current.click();
+        }
+    }, []);
+
+    return (
+        <button
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/transaction-modal/transaction-image.tsx:25]: common component I think

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**